Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -576,17 +572,9 @@ class BrowserViewController: UIViewController {
         self.present(alert, animated: true, completion: nil)
     }
 
-    fileprivate func shouldRestoreTabs() -> Bool {
+    fileprivate func canRestoreTabs() -> Bool {
         guard let tabsToRestore = TabManager.tabsToRestore() else { return false }
```

Comment:
Yes, but we fixed that with the `unarchiver.decodingFailurePolicy = .setErrorAndReturn` right? It would just return `nil`.